crawl-ref/source/webserver/templates/client.html

```diff
@@ -5,9 +5,10 @@
     <link rel="icon" href="/static/stone_soup_icon-32x32.png" type="image/png">
     <script type="text/javascript">
       var socket_server = "{{ socket_server }}";
+      var game_version = "{{ game_version }}";
     </script>
-    <script src="/static/scripts/contrib/require.js" data-main="/static/scripts/app"></script>
-    <link rel="stylesheet" type="text/css" href="/static/style.css">
+    <script src="/static/scripts/contrib/require.js?v={{ game_version }}" data-main="/static/scripts/app"></script>
+    <link rel="stylesheet" type="text/css" href="/static/style.css?v={{ game_version }}">
```

Comment:
Do we need to URI encode game_version here to? Maybe server.py could send it through already encoded?

Reply:
Actually in practise it doesn't needed as all chars in the version are URL-safe; did so in app.js purely out of habit